sdf.md

```diff
@@ -300,7 +300,7 @@ changes the state based on the current state of the Property named `value`.
 ## Elements of an SDF model
 
 The SDF language uses seven predefined Class Name Keywords for modeling connected
-Things, six of which are illustrated in {{fig-class-2}} (the seventh class `sdfProduct` is exactly like `sdfThing`).
+Things, six of which are illustrated in {{fig-class-2}} (the seventh class `sdfProduct` is similar to `sdfThing`).
```

Comment:
"similar to" is a bit weak.  In this figure, it would indeed look exactly like sdfThing.

Reply:
True, but at least the name is different :-) I'm afraid calling it "exactly like" raises the obvious question "why do you have it then" (which is answered later in the spec, but having forward reference here just for that sounds overkill).

Reply:
OK, will change this to say "would look exactly the same in the figure" or some such.